docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md

```diff
@@ -1,48 +1,158 @@
+```
+
+:::info Working outside a Jupyter Notebook
+
+In a Jupyter Notebook you will automatically see the results of the above code in a new cell when the code is run.  However, if you are working with a different Python interpreter you may need to explicitly print your results:
+
+```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py inspect_data_no_jupyter"
+```
+
+:::
+
+### 4. Use the Validator to create and run an Expectation
+
+The Validator provides access to all of our available Expectations as methods.  By running an `expect_*()` method from the Validator, two things will happen: The Validator will add the specified Expectation to an Expectation Suite (or edit an existing Expectation in the Expectation Suite, if applicable) in its configuration, and the specified Expectation will be run against the data we provided to the Validator with our Batch Request.
+
+```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py interactive_validation"
+```
+
+Since we are working in a Jupyter Notebook, the results of the Validation will be printed after we run an `expect_*()` method.  We can examine those results to determine if the Expectation needs to be edited (by running it again and providing additional optional parameters, or different parameter values).
+
+:::info Working outside a Jupyter Notebook
+As before, if you are not working in a Jupyter Notebook you may need to explicitly print your results:
+
+```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py interactive_validation_no_jupyter"
+```
+
+:::
+
+### 5. (Optional) Repeat step 4 to create additional Expectations or edit existing ones
+
+After reviewing the Validation Results that were returned when we created an Expectation, we can choose to edit that Expectation by running the `validator.expect_*()` method with different parameters than previously.  We can also have the Validator run an entirely different `expect_*()` method and create additional Expectations.  All the Expectations that we create will be stored in a list in the Validator's configuration.
+
+:::tip What if I want to use the same Expectation more than once?
+
+GX takes into account certain parameters when determining if an Expectation is being added to the list or if an existing Expectation should be edited.  For example, if you are using an expectation such as `expect_column_*()` you would edit an existing Expectation by providing the same `column` parameter when repeating the previous steps, and different values for any other parameters.  If you used a different value for the `column` parameter, you will create an additional instance of the Expectation for the new `column` value, rather than overwrite the Expectation you defined with the first `column` value.
```

Comment:
```suggestion
GX takes into account certain parameters when determining if an Expectation is being added to the list or if an existing Expectation should be edited.  For example, if you are using an expectation such as `expect_column_*()` you would edit an existing Expectation by providing the same `column` parameter when repeating the previous steps, and different values for any other parameters.  If you used a different value for the `column` parameter, you will create an additional instance of the Expectation for the new `column` value, rather than overwrite the Expectation you defined with the first `column` value.
```
 _when repeating the previous steps_ - What specific steps?

I think this needs some simplification and clarification